### DISCUSSION TITLE
Change search result text colors?

### DISCUSSION THREAD

**Discussion by cem-al** [wezterm/wezterm#2957] [Q&A] [answered]
Is it possible to style the results of a search (`cmd+f`)?

See the pink results below, would like to style those:
<img width="1532" alt="image" src="https://user-images.githubusercontent.com/60609268/212468006-63074aea-e417-4f85-8e80-ff37277be97a.png">

**Comment by wez** [comment, marked as answer]
Yeah, look for the `copy_mode_xxx` settings in the config fragment shown at https://wezfurlong.org/wezterm/config/appearance.html#defining-your-own-colors
(Search mode is really just a mode of copy mode)
